metricbeat/mb/module/runner.go

```diff
@@ -53,23 +55,37 @@ type Runner interface {
 }
 
 // NewRunner returns a Runner facade. The events generated by
+// the Module will be published to one of new publisher.Clients generated from the
+// pubClientFactory.
+func NewRunner(clients map[string]beat.Client, mod *Wrapper) Runner {
+	return &runner{
+		logger:  logp.NewLogger("runner"),
+		done:    make(chan struct{}),
+		mod:     mod,
+		clients: clients,
+	}
+}
+
+// NewRunnerForStaticModule returns a Runner facade. The events generated by
 // the Module will be published to a new publisher.Client generated from the
 // pubClientFactory.
-func NewRunner(client beat.Client, mod *Wrapper) Runner {
+func NewRunnerForStaticModule(client beat.Client, mod *Wrapper) Runner {
 	return &runner{
```

Comment:
Nit. Base this method in `NewRunner`, so it is easier to keep them updated if we change something in `runner`.
```
clients := map[string]beat.Client{staticModuleSink: client}
return NewClient(clients, mod)

Reply:
Refactored. Not relevant anymore.